notebooks/Janki_notebook_py.py

```diff
@@ -0,0 +1,73 @@
+
+"""Untitled1.py
+
+
+
+Original file is located at
+    https://colab.research.google.com/drive/114EwM4U9MTEpmfYlOUqvKQnr3s-u5-C2
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+df = pd.read_csv('merged_train_data.csv')
+df
+
+df['gender'].mean()
+
+df['gender'].median()
+
+df['gender'].std()
+
+df['gender'].value_counts()
+
+import matplotlib.pyplot as plt
+df.hist(column='gender', grid=False, edgecolor='black', color='red')
+
+"""There are more Females than Males."""
+
+df['bp_high'].mean()
+
+df['bp_high'].median()
+
+df['bp_high'].std()
+
+df['bp_high'].value_counts()
+
+import matplotlib.pyplot as plt
+
+df.hist(column='bp_high', grid=False, edgecolor='black', color = 'darkblue')
+
+df['bp_lo'].describe()
+
+import matplotlib.pyplot as plt
+
+df.hist(column='bp_lo', grid=False, edgecolor='black', color='purple')
+
+import pandas as pd
```

Comment:
Multiple Imports in the same file
```
import pandas as pd
import matplotlib.pyplot as plt
```
keep this two library imports on the top and remove other imports